### DISCUSSION TITLE
Jax equivalent of Scipy.constants h, c, k

### DISCUSSION THREAD

**Discussion by PoulomiPradhan** [jax-ml/jax#13179] [Q&A] [answered]
What Jax library I should import that is equivalent to below constants 
```python

from scipy.constants import h,c,k

```

**Comment by jakevdp** [comment, marked as answer]
You could use those constants directly; they are Python scalars and should work correctly with JAX code.

**Reply by PoulomiPradhan** [reply]
I didn't understand. How to import all those constants in Jax ? Also other mathematical functions like log10, root how to import those in a Jax file ? (I believe import math functions does not work for Jax , as they fail to do auto differentiation in Jax). As an example I am using jax.numpy for importing sin, sqrt and importing jax.lax for acos.

**Reply by jakevdp** [reply]
JAX supports python floats as inputs, so if you have constants defined as Python floats you can use them directly. For example:
```python
from scipy.constants import h,c,k
import jax.numpy as jnp
x = h * jnp.arange(10.0)
```
Regarding functions like `log10`, you should use the equivalent function defined in jax (for example, use `jax.numpy.log10` in place of `numpy.log10` or `math.log10`). If there is a function you need that is not available in JAX, feel free to open a feature request!

**Reply by PoulomiPradhan** [reply]
Thank you!
